pkg/admission/plugins.go

```diff
@@ -67,6 +68,7 @@ var AllOrderedPlugins = beforeWebhooks(kubeapiserveroptions.AllOrderedPlugins,
 	reservedcrdannotations.PluginName,
 	reservedcrdgroups.PluginName,
 	crdnooverlappinggvr.PluginName,
+	reservedmetadata.PluginName,
```

Comment:
how do we ensure that admission can set some of the annotation/labels, but the user cannot?

Reply:
as discussed OOB we'll tackle this part once front-proxy changes are merged.